blueos_repository/consolidate.py

```diff
@@ -96,7 +96,7 @@ async def fetch_readme(url: str) -> str:
 
     async def all_repositories(self) -> AsyncIterable[RepositoryEntry]:
         repos = self.repo_folder()
-        for repo in repos.glob("**/metadata.json"):
+        for repo in repos.glob("repos/**/metadata.json"):
```

Comment:
This seems like it should already be the base folder, so should probably be fixed in the `repo_folder` method rather than a hardcoded extra directory here.

Reply:
I agree, that makes more sense. I will split this change off and create two new pull requests